posthog-node/src/posthog-node.ts

```diff
@@ -9,13 +9,15 @@ import {
   PosthogFlagsAndPayloadsResponse,
   PostHogPersistedProperty,
 } from '../../posthog-core/src'
-import { PostHogMemoryStorage } from '../../posthog-core/src/storage-memory'
+import { PostHogMemoryStorage } from '../../posthog-core/src/storage/storage-memory'
+import { PostHogFsStorage } from '../../posthog-core/src/storage/storage-fs'
 import { EventMessageV1, GroupIdentifyMessage, IdentifyMessageV1, PostHogNodeV1 } from './types'
 import { FeatureFlagsPoller } from './feature-flags'
 import { fetch } from './fetch'
+import { PostHogStorage } from 'posthog-core/src/storage/storage'
 
 export type PostHogOptions = PosthogCoreOptions & {
-  persistence?: 'memory'
+  persistence?: string
```

Comment:
Let's keep this a literal with possible options, and add another configuration option to pass in the file path

Reply:
How's the new interface look?

Reply:
looks good!